### ISSUE TITLE
Can not install with pip

### ISSUE THREAD

**Issue by thomasfillon** [DavidDoukhan/py_sonicvisualiser#1]
Hi David !

Trying to install py_sonicvisualiser through pip raise the following Error:

```
 $ sudo pip install py_sonicvisualiser
 Downloading/unpacking py-sonicvisualiser
  Running setup.py egg_info for package py-sonicvisualiser
    Traceback (most recent call last):
      File "<string>", line 14, in <module>
      File "/home/thomas/code/timeside/TimeSide/build/py-sonicvisualiser/setup.py", line 37, in <module>
        long_description = open('README.md').read(),
    IOError: [Errno 2] No such file or directory: 'README.md'
    Complete output from command python setup.py egg_info:
   Traceback (most recent call last):

  File "<string>", line 14, in <module>

 File "/home/thomas/code/timeside/TimeSide/build/py-sonicvisualiser/setup.py", line 37, in <module>

   long_description = open('README.md').read(),

IOError: [Errno 2] No such file or directory: 'README.md'
```

**Comment by DavidDoukhan** [comment]
Hi Thomas,
Thanks for the report
This issue should have been fixed for versions above 0.2-9
Let me know if you still experience installation problems with this version.

**Comment by thomasfillon** [comment]
Everything is fine now !
Thank you David !
